package.json

```diff
@@ -5,6 +5,25 @@
   "activationEvents": [
     "*"
   ],
+  "contributes": {
+    "actions": [
+      {
+        "id": "langserver.status",
+        "actionItem": {
+          "description": "${get(context, `langserver.statusDescription`)}",
```

Comment:
You can hardcode the iconURL because it never needs to change.

And the description can be:

```
Code intelligence active for ${resource.language}
```

https://sourcegraph.com/github.com/sourcegraph/sourcegraph-extension-api/-/blob/src/client/context/context.ts#L70

And you can remove the entire ` sourcegraph.workspace.onDidOpenTextDocument.subscribe(...)` call.

This is better because the contribution will show up more quickly b/c it's not dynamic.